lib/lookup.json

```diff
@@ -283,7 +283,7 @@
     "maintainers": "broofa"
   },
   "serialport": {
-    "flaky": ["ppc", "win32"],
+    "flaky": ["ppc", "win32", "ubuntu"],
```

Comment:
this should be followed up with serialport before adding to flaky

/cc @reconbot